examples/browser-webrtc/src/lib.rs

```diff
@@ -24,7 +24,9 @@ pub async fn run(libp2p_endpoint: String) -> Result<(), JsError> {
             webrtc_websys::Transport::new(webrtc_websys::Config::new(&key))
         })?
         .with_behaviour(|_| ping::Behaviour::new(ping::Config::new()))?
-        .with_swarm_config(|c| c.with_idle_connection_timeout(Duration::from_secs(5)))
+        // Ping does not KeepAlive, so we set the idle connection timeout to 32_212_254u64,
```

Comment:
Why such a large value? Isn't 30 seconds entirely sufficient to see that the connection works?

Reply:
Part of this was trying to determine / document why there was a strange idle limit in wasm in case someone tries to do `u64::MAX` like I initially tried to do. For the example though I made some changes which clear it up